tests/unittest/llmapi/test_llm_pytorch.py

```
@@ -288,6 +291,56 @@ def _check_contains_expected_message(stdout: str, stderr: str):
+    with pytest.raises(RuntimeError):
+        check_llama_7b_multi_lora_from_request_test_harness(
+            LLM,
+            lora_config=lora_config_no_cache_size_values,
+            peft_cache_config=PeftCacheConfig(device_cache_percent=0.000001),
+            # Disable CUDA graph
+            # TODO: remove this once we have a proper fix for CUDA graph in LoRA
+            cuda_graph_config=None)
+
+
+def test_llama_7b_lora_config_overrides_peft_cache_config():
+    """Tests that cache size args in lora_config LLM arg override the cache size parameters in peft_cache_config LLM arg."""
```

Ruff (0.12.2)

Line too long (124 > 120)

(E501)